ds3-sdk/src/test/java/com/spectralogic/ds3client/helpers/FileObjectPutter_Test.java

```diff
@@ -56,27 +65,47 @@ public void testSymlink() throws IOException {
     }
 
     @Test
-    public void testRegularFile() throws IOException {
-        final Path tempDir = Files.createTempDirectory("ds3_file_object_putter_");
+    public void testRelativeSymlink() throws IOException, URISyntaxException {
+        assumeThat(System.getProperty("os.name"), not(containsString("Windows")));
```

Comment:
We are starting to use this windows check more often, we should probably create a util function for isWindows.
I did it in the cli